source/opt/cfg_cleanup_pass.cpp

```diff
@@ -276,7 +253,7 @@ Pass::Status CFGCleanupPass::Process(ir::Module* module) {
   // Process all entry point functions.
   ProcessFunction pfn = [this](ir::Function* fp) { return CFGCleanup(fp); };
   bool modified = ProcessReachableCallTree(pfn, module);
-  FinalizeNextId(module_);
+  FinalizeNextId();
```

Comment:
Is there a reason we don't get rid of FinalizeNextId() and just incorporate into asking for a new id?  I don't see a benefit of having it one its own, and it could be a source of error if someone forgets to call it.

Reply:
I'm ok deferring this.

I filed https://github.com/KhronosGroup/SPIRV-Tools/issues/929 for this refactor.

Reply:
D'oh!  Of course.  Done.